include/rosdiscover-clang/ApiCall/Calls/SubscribeTopicCall.h

```diff
@@ -15,6 +15,24 @@ class SubscribeTopicCall : public NodeHandleRosApiCall {
   RosApiCallKind const getKind() const override {
     return RosApiCallKind::SubscribeTopicCall;
   }
+  
+  Callback* getCallback(clang::ASTContext &context) const override {
+    auto *callExpr = getCallExpr();
+    auto numArgs = callExpr->getNumArgs();
+
+    llvm::outs() << "[SubscribeTopicCall] Finding Callback (" << numArgs << ")\n";
+
+    // if the call only has one argument, then we don't know what the callback is for now
+    if (numArgs < 3) {
```

Comment:
Print a warning if this is the case so that we're aware of inadequacies in the analysis 👍🏻